packages/flame_fire_atlas/lib/flame_fire_atlas.dart

```diff
@@ -21,13 +22,27 @@ extension FireAtlasExtensions on Game {
   }
 }
 
+/// {@template _selection}
+/// Represents the basic information of a selection inside a fire atlas file,
+/// containing its [id] and its coordinates and dimensions.
+/// {@endtemplate}
 class Selection {
+  /// The id of the selection
   String id;
+
+  /// The horizontal coordinate of the selection
   int x;
+
+  /// The vertical coordinate of the selection
   int y;
+
+  /// The width of the selection
   int w;
```

Comment:
Not related to this PR, but these should probably be `width` and `height` to follow our own and Flutter's convention.

Reply:
This are all internal, I kept a single letter, so on the generated file, we have less characters and a smaller file.

But we can change the name of the field on the code model later.

Reply:
I think you can change the json name with an annotation (like gson)

Reply:
(no strong opinion about naming though)
also if you want to reduce file size json is not the best option, something like protobuff or just a binary serialization would be smaller. or just gzip the content :P

Reply:
We do gzip the content after, but wanted to make it even small, I know it will not make it much smaller, but even so.

Reply:
This will be a matter of bytes, and the file is only loaded once? That kind of difference in file size really shouldn't matter on modern devices.

Reply:
I know it would make much difference, by since it is possible why not? That was the reasoning for the choice in the past.

Either way, since from the beginning it was know that this could be improved. Eventually @wolfenrain lib could be used here instead for optimal file sizes.